.circleci/dockerfile

```diff
@@ -0,0 +1,12 @@
+# adapted from https://github.com/hypertrace/helm-gcs-packager/blob/master/Dockerfile
+# manually built and pushed as praqma/helm-gsutil
+FROM google/cloud-sdk:alpine
+
+# Install Helm
+RUN curl -Lo helm-v3.1.2-linux-amd64.tar.gz https://get.helm.sh/helm-v3.1.2-linux-amd64.tar.gz && \
```

Comment:
this installs helm v3.1.2 , but the script uses helm v2.X commands. You can either update the script to use helm v3 commands or install helm v2 here